src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {

```

Comment:
Don't use raw type, parametrize it

Reply:
I don't agree with that

Reply:
You should parametrize it. Your concrete producer should be parametrize. Fix it for all producers 
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```